azure-pipelines.yml

```diff
@@ -79,6 +79,11 @@ jobs:
             -
                 script: "python -m pip install --upgrade pip && pip install -r test_requirements.txt"
                 displayName: "Install mbed-os test dependencies"
+            -
+                script: "pip install lxml"
+                continueOnError: true
```

Comment:
Is it valid to continue if this fails?

Reply:
It is valid. I removed this to avoid any confusion